src/core/config/openthread-core-config-check.h

```diff
@@ -41,6 +41,16 @@
 #ifndef OPENTHREAD_CORE_CONFIG_CHECK_H_
 #define OPENTHREAD_CORE_CONFIG_CHECK_H_
 
+/**
+ * @addtogroup config-openthread-core
+ *
+ * @brief
+ *   This module include configuration variables for Openthread core configuration checks.
+ *
+ * @{
+ *
+ */
```

Comment:
Not sure we need to add the `core-config-check` in documentaion. Suggest removing this.

This header is intended as a place to add checks to ensure any removed/renamed config is no longer.

Reply:
removed it, thanks